### PR TITLE
Update document

### PR DESCRIPTION
**Pull request by clyang82**
ref: https://issues.redhat.com/browse/ACM-4025

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -63,24 +77,19 @@ Both the global hub manager and grafana services need a postgres database to col
...
-![cluster overview](cluster_overview.png)
 ### Import a reginal hub cluster
-#### Import a regional hub cluster in default mode
+#### Import a regional hub cluster in default mode (tech preview)
 It requires to disable the cluster self management in the existing ACM hub cluster. Set `disableHubSelfManagement=true` in the `multiclusterhub` CR to disable automatic import of the hub cluster as a managed cluster.
 
 After that, follow the [Import cluster](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#importing-a-cluster) document to import the regional hub cluster.
 
-#### Import a regional hub cluster in hosted mode
+#### Import a regional hub cluster in hosted mode (tech preview)
```
Comment (hchenxa) [inline doc/README.md:86]: this should not be the Tech preview in this release.

File: doc/dev-preview.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+### Create a regional hub cluster (dev preview)
+Refer to the original [Create cluster](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#creating-a-cluster) document to create the managed cluster in the global hub cluster. add labels of `global-hub.open-cluster-management.io/hub-cluster-install: ''` in managedcluster CR and then the new created managed cluster can be switched to be a regional hub cluster automatically. In other words, the latest released RHACM is installed in this managed cluster. You can get the ACM hub information in the cluster overview page.
```
Comment (hchenxa) [inline doc/dev-preview.md:2]: 2.6 was not supported. prefer to use 2.7+ doc link here.

File: doc/dev-preview.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+![cluster overview](cluster_overview.png)
+### Create a global policy (dev preview)
+You can navigate to `Governance` from the navigation menu, and then click the `Create policy` to create a global policy. The `policyset` is not supported yet.
```
Comment (hchenxa) [inline doc/dev-preview.md:5]: the grc was disabled by default on global hub cluster. so how to create the policy here?

File: doc/dev-preview.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+![create global policy](create_policy.png)
+If you want to create a global policy which can be propagated to regional hub clusters by transport, you need to add `global-hub.open-cluster-management.io/global-resource=""` manually for the Policy/PlacementRule/PlacementBinding.
+### Create a global application (dev preview)
+You can navigate to `Applications` from the navigation menu, and then click the `Create application` to select `Subscription`. The `ApplicationSet` is not supported yet.
```
Comment (hchenxa) [inline doc/dev-preview.md:9]: same as grc, the app will be disabled by default in global hub cluster

File: doc/README.md [hunk 5]
```diff
@@ -37,22 +43,30 @@ Grafana runs on the global hub cluster as the main service for Global Hub Observ
...
+Both the global hub manager and grafana services need a postgres database to collect and display data. The data can be accessed by creating a storage secret, which contains the following two fields:
 - `database_uri`: Required, the URI user should have the permission to create the global hub database in the postgres.
 - `ca.crt`: Optional, if your database service has TLS enabled, you can provide the appropriate certificate depending on the SSL mode of the connection. If the SSL mode is `verify-ca` and `verify-full`, then the `ca.crt` certificate must be provided.
 
+#### Transport secret
+Right now, we support Kafka transport only. You need to create a secret for the Kafka transport. The secret contains the following fields:
+- `bootstrap.servers`: Required, the Kafka bootstrap servers.
+- `ca.crt`: Optional, if your Kafka service has TLS enabled, you can provide the appropriate certificate depending on the SSL mode of the connection. If the SSL mode is `verify-ca` and `verify-full`, then the `ca.crt` certificate must be provided.
```
Comment (yanmxa) [inline doc/README.md:53]: As far as I know, there is no SSL mode to set `verfiy-ca` or `verfy-full` in kafka configuration.

File: doc/README.md [hunk 6]
```diff
@@ -37,22 +43,30 @@ Grafana runs on the global hub cluster as the main service for Global Hub Observ
...
 ### Installation
 
-#### Install the multicluster global hub operator from OpenShift console
+#### 1. [Install the multicluster global hub operator on a disconnected environment](./disconnected_environment/README.md)
+
+#### 2. Install the multicluster global hub operator from OpenShift console
 
 1. Log in to the OpenShift console as a user with cluster-admin role.
 2. Click the Operators -> OperatorHub icon in the left navigation panel.
 3. Search for the `multicluster global hub operator`.
```
Comment (bjoydeep) [inline doc/README.md:65]: This is misleading now. We should not recommend users to do this till the operator code is community updated regularly.

File: doc/README.md [hunk 7]
```diff
@@ -63,24 +77,19 @@ Both the global hub manager and grafana services need a postgres database to col
 
 > Note: the multicluster global hub is available for x86 platform only right now.
 
-
-### Create a regional hub cluster
-Refer to the original [Create cluster](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html/multicluster_engine/multicluster_engine_overview#creating-a-cluster) document to create the managed cluster in the global hub cluster. add labels of `global-hub.open-cluster-management.io/hub-cluster-install: ''` in managedcluster CR and then the new created managed cluster can be switched to be a regional hub cluster automatically. In other words, the latest released RHACM is installed in this managed cluster. You can get the ACM hub information in the cluster overview page.
-![cluster overview](cluster_overview.png)
 ### Import a reginal hub cluster
```
Comment (bjoydeep) [inline doc/README.md:80]: typo - regi`o`nal

File: doc/README.md (hunk 1 above)
Comment (bjoydeep) [reply doc/README.md:86]: agreed

File: doc/dev-preview.md (hunk 2 above)
Comment (bjoydeep) [reply doc/dev-preview.md:2]: Not sure what is the relevance of this. We do not support/advertise capability in 2.7

File: doc/dev-preview.md (hunk 3 above)
Comment (bjoydeep) [reply doc/dev-preview.md:5]: Again, we do not support this in TP. This is old documentation from dev preview. Should be removed.

File: doc/dev-preview.md (hunk 4 above)
Comment (bjoydeep) [reply doc/dev-preview.md:9]: This is old documentation from dev preview. Should be removed.

File: doc/dev-preview.md (hunk 3 above)
Comment (clyang82) [reply doc/dev-preview.md:5]: Done

File: doc/README.md (hunk 6 above)
Comment (clyang82) [reply doc/README.md:65]: Yes. we update the operator regularly. Once the API is changed, we will have a PR to update in community operatorhub. in the develop stage, it is pre-release. once it is released officially, will have a release in the community as well.

File: doc/README.md (hunk 5 above)
Comment (clyang82) [reply doc/README.md:53]: Do you know how to configure SSL mode in kafka? it should be in kafka broker. I cannot find `SSLMode` or `verify-ca` keywords in their code base. How do they achieve it?

File: doc/README.md [hunk 8]
```diff
@@ -63,152 +79,17 @@ Both the global hub manager and grafana services need a postgres database to col
...
-# modify the service to LoadBalancer, default port is 5432
-kubectl patch postgrescluster hoh -n hoh-postgres -p '{"spec":{"service":{"type":"LoadBalancer"}}}'  --type merge
-# host/ user/ password/ database
-kubectl get svc -n hoh-postgres hoh-ha -ojsonpath='{.status.loadBalancer.ingress[0].hostname}'
-kubectl get secrets -n hoh-postgres hoh-pguser-postgres -o go-template='{{index (.data) "user" | base64decode}}'
-kubectl get secrets -n hoh-postgres hoh-pguser-postgres -o go-template='{{index (.data) "password" | base64decode}}'
-kubectl get secrets -n hoh-postgres hoh-pguser-postgres -o go-template='{{index (.data) "dbname" | base64decode}}'
-```
\ No newline at end of file
+### Grafana dashboards
```
Comment (hchenxa) [inline doc/README.md:95]: do we need this? seems no content for this section.

File: doc/README.md [hunk 9]
```diff
@@ -63,152 +79,17 @@ Both the global hub manager and grafana services need a postgres database to col
...
-    addon.open-cluster-management.io/hosting-cluster-name: local-cluster
-spec:
-  installNamespace: open-cluster-management-hub1-addon-workmanager
-EOF
-```
-You have to create a kubeconfig secret for the work-manager add-on via the following command:
-```
-oc create secret generic work-manager-managed-kubeconfig --from-file=kubeconfig=<your regional hub kubeconfig> -n open-cluster-management-hub1-addon-workmanager
-```
+After that, follow the [Import cluster](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html-single/clusters/index#importing-a-target-managed-cluster-to-the-hub-cluster) document to import the regional hub cluster.
```
Comment (hchenxa) [inline doc/README.md:87]: should we add checking the globalhub-addon status in this part after the cluster was imported?

File: doc/README.md (hunk 8 above)
Comment (clyang82) [reply doc/README.md:95]: @dislbenn will add later

**Pull request merged**